packages/block-library/src/media-text/edit.native.js

```diff
@@ -165,21 +166,22 @@ class MediaTextEdit extends Component {
 			onClick: () => setAttributes( { mediaPosition: 'right' } ),
 		} ];
 
-		const onVerticalAlignmentChange = ( alignment ) => {
+		/* const onVerticalAlignmentChange = ( alignment ) => {
 			setAttributes( { verticalAlignment: alignment } );
-		};
+		}; */
 
 		return (
 			<>
 				<BlockControls>
 					<Toolbar
 						controls={ toolbarControls }
 					/>
+					{ /* // Temporarily commenting out until alignment functionality is fixed
 					<BlockVerticalAlignmentToolbar
 						onChange={ onVerticalAlignmentChange }
 						value={ verticalAlignment }
 						isCollapsed={ false }
-					/>
+					/> */ }
```

Comment:
I was thinking if we would benefit from using a dev flag instead of commenting out the code.
Alternatively we could actually remove the not-needed code and then revert this commit.

But since it's a hot-fix and this seems to be intended to be worked on right away after the release (I might be wrong :) I think it's OK to merge as it is. 👍

Reply:
Yes we are going to revisit this.

__DEV__ flag sounds good but it requires much complex testing effort like generating bundles and opening PRs on parent apps. 

IMHO removing is better, but we'll try bringing this back in a few weeks so not sure it'll worth it, and in that case bringing this code back would generate much more diff. But according to what happens next we should definitely revisit this.